src/commands/up.rs

```diff
@@ -235,7 +249,7 @@ impl UpCommand {
             })?;
         }
 
-        let status = child.wait()?;
+        let status = tokio::task::spawn(async move { child.wait() }).await??;
```

Comment:
nit: Would it be more natural to change this function to use a `tokio::process::Command`, so that you merely had to await `child.wait()` rather than explicitly spawn a task and awaiting that?

Reply:
That would be a lot better, had I known that existed :) This is pretty much my first serious venture into Rust. I'll fix.